summarization/extractor_output_preprocessor.py

```diff
@@ -50,13 +51,20 @@ class ExtractorOutputPreprocessor:
     # we can display as title so it is readable
     # and still does not block out other content
     MAX_TITLE_LENGTH = 100
+    EMBEDDED_CONTENT_TYPES = [
+        ContentType.EMBEDDED_INSTAGRAM_POST,
+        ContentType.EMBEDDED_PINTEREST_PIN,
+        ContentType.EMBEDDED_TWEET,
+        ContentType.EMBEDDED_YOUTUBE_VIDEO
+    ]
```

Comment:
Instead of this, you should add method in enum
https://docs.python.org/3/library/enum.html#allowed-members-and-attributes-of-enumerations

Reply:
Updated accordingly.